tests/unittests/QuantizationTest.cpp

```diff
@@ -333,6 +333,58 @@ TEST(Quantization, enableRowwiseQuantizedFullyConnected) {
   EE.run(bindings);
 }
 
+/// Check that SLWS is correctly fused rowwise-quantized by the quantizer.
+TEST(Quantization, enableRowwiseQuantizedSLWS) {
+  ExecutionEngine EE;
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+  PlaceholderBindings bindings;
+
+  auto *data = mod.createPlaceholder(ElemKind::FloatTy, {3, 1}, "data", false);
+  auto *weights =
+      mod.createPlaceholder(ElemKind::FloatTy, {8}, "weights", false);
+  auto *indices =
+      mod.createPlaceholder(ElemKind::Int64ITy, {8}, "indices", false);
+  auto *lengths =
+      mod.createPlaceholder(ElemKind::Int32ITy, {4}, "lengths", false);
+
+  bindings.allocate(data)->init(Tensor::InitKind::Xavier, 3, mod.getPRNG());
```

Comment:
nit: we do not necessarily need to `init` this data (we already kinda profile it by having `std::vector<NodeQuantizationInfo>`.